04week/tomorrows-technology/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
```

Comment:
The side seems to always have a scrollbar at the bottom. 
I think there is some slight mistake on the width of the page